### PR TITLE
Feature/wysiwyg editors

### PR DESCRIPTION
**Pull request by germanbisurgi**
Alternatives for CKEditor: Jodit and SCEditor
Markdown editor: simpleMDE

### CODE REVIEW

**Comment by germanbisurgi** [comment]
editors-configs

File: src/JoditAsset.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * file that was distributed with this source code.
+ */
+
+namespace dmstr\jsoneditor;
+
+use yii\web\AssetBundle;
+
+class JoditAsset extends AssetBundle
+{
+    public $sourcePath = '@dmstr/jsoneditor/assets/';
```
Comment (eluhr) [inline src/JoditAsset.php]: Use @dmstr/jsoneditor/assets/jodit here to prevent publishing unwanted files

Comment (eluhr) [reply src/JoditAsset.php]: Or why do we not use the files in  `/assets/jodit`

File: src/SceditorAsset.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+namespace dmstr\jsoneditor;
+
+use yii\web\AssetBundle;
+use yii\web\JqueryAsset;
+
+class SceditorAsset extends AssetBundle
+{
+    public $sourcePath = '@dmstr/jsoneditor/assets/sceditor/minified';
```
Comment (eluhr) [inline src/SceditorAsset.php]: why do we not use the files in /assets/sceditor/minified?

File: src/JsonEditorWidget.php [hunk 3]
```diff
@@ -125,12 +154,76 @@ public function init()
...
+              "hidePoweredByJodithidePoweredByJodit": true,
+              "spellcheckspellcheck": false,
+              "controls": {
+                "classSpan": {
+                  "list": {}
+                }
+              }
+            }';
+
+            $json = Yii::$app->settings->getOrSet('jodit.config', $joditConfig, 'jsoneditor', 'object');
```
Comment (eluhr) [inline src/JsonEditorWidget.php]: Settings should be optional - see: https://github.com/dmstr/yii2-json-editor/pull/13

**Review by eluhr — CHANGES_REQUESTED** [review]

**Comment by eluhr** [comment]
Why are the assets bundles not in the `assets` directory?

**Comment by germanbisurgi** [comment]
Ping

**Review by eluhr — APPROVED** [review]
LGTM

**Review by schmunk42 — CHANGES_REQUESTED** [review]
Please include the LICENSE files and a note in the README about the bundled assets.

**Review by schmunk42 — APPROVED** [review]

**Comment by germanbisurgi** [comment]
This pull request is no longer WIP. Should i merge and tag?

**Comment by schmunk42** [comment]
Yes, go ahead.

**Pull request merged**